Parent/Parent/Create Account/CreateAccountViewController.swift

```diff
@@ -89,16 +88,19 @@ class CreateAccountViewController: UIViewController, ErrorViewController {
     }
 
     func setupKeyboardNofications() {
-        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillChangeFrame(_:)), name: UIResponder.keyboardWillShowNotification, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillChangeFrame(_:)), name: UIResponder.keyboardWillChangeFrameNotification, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillChangeFrame(_:)), name: UIResponder.keyboardWillHideNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow(_:)), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide(_:)), name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+    @objc func keyboardWillHide(_ notification: Notification) {
+        scrollPadding.constant = 0
     }
 
-    @objc func keyboardWillChangeFrame(_ notification: Notification) {
+    @objc func keyboardWillShow(_ notification: Notification) {
         guard
             let info = notification.userInfo as? [String: Any],
             let keyboardFrame = info[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect
             else { return }
+        scrollPadding.constant = 200
```

Comment:
I'm not able to scroll to the bottom of the form with the keyboard open. I'm guessing this isn't enough spacing.